fuzz-proto/fuzz-v1.asn

```diff
@@ -0,0 +1,72 @@
+-- Specification for Fuzzer Protocol V1
+
+FuzzProtocol DEFINITIONS ::= BEGIN
+
+-- `Block` and `Header` as defined by the gray paper
+-- Refer to jam-types.asn in the jam-test-vectors repo
+
+U8 ::= INTEGER (0..255)
+U32 ::= INTEGER (0..4294967295)
+
+Hash ::= OCTET STRING (SIZE(32))
+HeaderHash ::= Hash
+StateRootHash ::= Hash
+
+-- Features is a u32 where the MSB (bit 31) is reserved for future extensions
+-- If MSB is 1, another u32 follows with more feature flags
+Features ::= U32
+
+-- Feature bit constants
+feature-block-ancestry INTEGER ::= 1      -- 2^0
+feature-simple-forking INTEGER ::= 2      -- 2^1
+feature-extension INTEGER ::= 2147483648  -- 2^31 = 0x80000000
+
+Version ::= SEQUENCE {
+    major U8,
+    minor U8,
+    patch U8
+}
+
+PeerInfo ::= SEQUENCE {
+    -- Fuzzer protocol version
+    fuzz-version U8,
```

Comment:
Do we need a full version tripled for this? Will the fuzzer apply some SemVer rules or require exact match of the latest version?